nf_requirements/v1alpha1/interface_interfaces.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2023 Nephio.
```

Comment:
nit: The Nephio Authors

Reply:
I changed on all the files as this was not there originally.